tests/test_smoke.py

```diff
@@ -1052,11 +1061,11 @@ def test_private_bucket(self, private_bucket):
             storage_obj = storage_lib.Storage(source=private_bucket)
 
     @staticmethod
-    def cli_ls_cmd(store_type, bucket_name):
+    def cli_ls_cmd(store_type, bucket_name, suffix=''):
         if store_type == storage_lib.StoreType.S3:
-            return ['aws', 's3', 'ls', f's3://{bucket_name}']
+            return ['aws', 's3', 'ls', f's3://{bucket_name}{suffix}']
```

Comment:
It is a bit awkward have an unspecified requirement for the suffix after the bucket name. I.e. is the suffix `/root_path` or `root_path`?

Reply:
Thanks! I've made it more explicit now